pages/docs/literature-guide/literature-guide-overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Literature guide
+permalink: literature-guide-overview.html
+keywords: literature, papers, numerics
+summary: "Theses and papers for theoretical background of preCICE"
+---
+
+Wherever the information in this documentation is not enough (in this case, [let us know](https://www.precice.org/resources/#contact)), you may find a wide spectrum of additional information in [publications](https://www.precice.org/publications/). This page will guide you through it.
```

Comment:
Two points that I currently don't know how to handle (@uekerman):
1. Where do we now link for the "contact us"? Maybe the top-level "community" page? Maybe "community/channels"? Inany case, we don't have this already. It would also make sense to just merge this as it is and then have a second iteration where we check/update all the links.
2.  Where do we link for the publications? From our [proposed structure](https://pad.gwdg.de/jxoCHHtJSYiZt6hS3XgPVA) I understand that in the community we put the "everything that cites preCICE". Should we add here a subpage with our publications?

Reply:
concerning 1.
I think "community/channels" is good. For the moment you could simply put `community-channels.html`. We have to check all links in the end anyway. Yes, let's not make this a merge stopper.

concerning 2.
Yes, in our proposed structure, we put everything that cites preCICE to community. I guess our thinking was that the "literature overview" is indeed where we list all publications about preCICE. So I guess, we don't need another page there. Right?

Reply:
> I guess our thinking was that the "literature overview" is indeed where we list all publications about preCICE. So I guess, we don't need another page there. Right?

But still, [our current list of publications](https://www.precice.org/publications/) already has >50 entries. We probably want to organize this a bit better. We could merge this now and decide on what to do with the publications later.